lang/sql/sql.talon

```diff
@@ -0,0 +1,38 @@
+mode: command
+and mode: user.sql
+mode: command
+and mode: user.auto_lang
+and code.language: sql
+-
+tag(): user.code_generic
+tag(): user.code_operators
+
+# Some commands have spaces around them to avoid having to continually say 'space'.
```

Comment:
I haven't done SQL in forever, so I don't remember anything. But if you need a variety of space options, it may be prudent to throw these in a list.

Then you could do things like...
```
pad <sql_list>: " {user.sql} "
```

Reply:
The problem I see with that approach is that it would be way more cumbersome to dictate the queries.